MatrixKit/Categories/NSData+MatrixKit.h

```diff
@@ -26,4 +26,14 @@
  */
 - (NSString*)MD5;
 
+/**
+ Return a SHA256 hash code from NSData content
+ */
+- (NSString*)SHA256;
+
+/**
+ Return a readable SHA256 hash code from NSData content
```

Comment:
Could be a little clearer on what the difference is between these two: the first is also a hex string (the second just has spaces)